wolox-training/rof-angular
# Implemented field edit on profile

## Summary

Implemented field edit on profile

## Test Plan

* Log your account or create a new one via register
* Go to profile on header
* Click edit fields button
* Make changes
* Accept them
* Refresh page to confirm data has chaned.

## Trello Card 

https://trello.com/c/1skrk0rU

## Screenshot

![changes](https://cloud.githubusercontent.com/assets/2911971/10678328/17d4ca42-78e8-11e5-921c-a23c2ad4b25f.png)
